README.md

```diff
@@ -27,6 +27,7 @@ Thanks to all [contributors](https://github.com/rulikkk/awesome-power-bi/graphs/
 * [Courses](#courses)
 * [Communities](#communities)
 * [Tutorials](#tutorials)
+* [Reviewers](#reviewers)
```

Comment:
> 

Used _leaders_ instaed of _reviewers_.
Added a header _Independent powerBI technology leaders_

Reply:
> Definitely not "reviewers" ... they look more like independent powerBI technology (or community) leaders / ambassadors / influencers. Need to pick a more respectful word =)

Used leaders instaed of reviewers.
Added a header Independent powerBI technology leaders

Reply:
> Definitely not "reviewers" ... they look more like independent powerBI technology (or community) leaders / ambassadors / influencers. Need to pick a more respectful word =)

Used _leaders_ instead of _reviewers_.
Addede an h4 header _Independent powerBI technology leaders_